# Engine id
string id

# Path to profile file in the file system that DecisionService runs
string file
---
# Success == 1, Failure == 0
int32 success
